server/templates/obslist.html

```diff
@@ -64,20 +64,34 @@ <h3>Observations</h3>
         <th><abbr title="Time of closest approach">TCA</abbr></th>
         <th><abbr title="Loss of Signal">LOS</abbr></th>
         <th>Satellite</th>
+        <th><abbr title="Two Line Element">TLE</abbr></th>
         <th>Station</th>
         <th>Image</th>
     </tr>
     {% for obs in obslist %}
         <tr>
             <td><a href="/obs/{{ obs.obs_id }}">{{ obs.obs_id }}</a></td>
-            <td>{{ obs.aos }}</td>
-            <td>{{ obs.tca }}</td>
-            <td>{{ obs.los }}</td>
+            <td>{{ obs.aos.isoformat(" ", "seconds") }}</td>
+            <td>{{ obs.tca.isoformat(" ", "seconds") }} {% if obs.aos == obs.tca %}
+                <abbr title="TCA is equal to AOS, which looks suspicious. Corrected rough estimate is shown below in parentheses.">
+                    <span class="glyphicon text-warning glyphicon-question-sign"></span></abbr>
+                <br/>({{ (obs.aos + (obs.los - obs.aos)/2).isoformat(" ", "seconds") }})
+                {% endif %}
+            </td>
+            <td>{{ obs.los.isoformat(" ", "seconds") }}</td>
             <td>
                 <a href="https://www.n2yo.com/satellite/?s={{ obs.sat_id }}">
                     {{ obs.sat_name }}
                 </a>
             </td>
+            <td>
+                {% if obs.tle %}
+                <span class="tle glyphicon text-success glyphicon-ok" data-toggle="tooltip" data-html="true" boundary="window"
+                      title="<b>TLE</b><br/>{{ obs.tle[0]}}<br/>{{ obs.tle[1] }}"></span>
```

Comment:
Why you place the TLE in tooltip? It takes up a lot of space and isn't human-readable. When TLE is in tooltip user cannot copy it too. I think that icon with TLE status is enough on observation list.

Reply:
Ok, removed.